src/pages/RepoPage/FailedTestsTab/GitHubActions/GitHubActions.tsx

```diff
@@ -0,0 +1,211 @@
+        with:
+          python-version: 3.11
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Test with pytest
+        run: |
+          pytest --cov --junitxml=junit.xml
+      # Copy and paste the codecov/test-results-action here
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v4
+        with:
+          token: \${{ secrets.CODECOV_TOKEN }}
+      - name: Upload test results to Codecov
+        if: \${{ !cancelled() }}
+        uses: codecov/test-results-action@v1
+        with:
+          token: \${{ secrets.CODECOV_TOKEN }}`
+
+function Step2() {
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  return (
+    <div>
+      <Card>
+        <Card.Header>
+          <Card.Title size="base">
+            Step 2: Add the script{' '}
+            <code className="rounded-md border border-ds-gray-secondary bg-ds-gray-primary p-1 text-sm">
+              codecov/test-results-action@v1
+            </code>{' '}
+            to your CI YAML file.
+          </Card.Title>
+        </Card.Header>
+        <Card.Content className="flex flex-col gap-4">
+          <p>
+            In your CI YAML file, add below scripts to the end of your test run.
+          </p>
+          <CodeSnippet clipboard={Step2Script}>{Step2Script}</CodeSnippet>
+          <p>
+            This action will download the Codecov CLI, and upload the{' '}
+            <span className="text-codecov-code">junit.xml</span> file generated
+            in the previous step to Codecov.
+          </p>
+        </Card.Content>
+      </Card>
+      <ExpandableSection className="mt-[-1px]">
+        <ExpandableSection.Trigger
+          isExpanded={isExpanded}
+          onClick={() => setIsExpanded(!isExpanded)}
+        >
+          <p className="font-normal">
+            Your final <span className="text-codecov-code">ci.yaml</span> file
+            for a project using{' '}
+            <span className="text-codecov-code">pytest</span> could look
+            something like this:
+          </p>
+        </ExpandableSection.Trigger>
+        <ExpandableSection.Content>
+          <CodeSnippet clipboard={JobsScript}>{JobsScript}</CodeSnippet>
+        </ExpandableSection.Content>
+      </ExpandableSection>
+    </div>
+  )
+}
+
+function Step3() {
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  return (
+    <div>
+      <Card>
+        <Card.Header>
+          <Card.Title size="base">Step 3: Run your test suit</Card.Title>
+        </Card.Header>
+        <Card.Content className="flex flex-col gap-4">
+          <p>
+            You can inspect the workflow logs to see if the call to Codecov
+            succeeded.
+          </p>
+          <img src={testsRunning.toString()} alt="GitHub Actions Tests" />
+          <p>
+            Run your tests as usual. You need to fail some tests to view the
```

Comment:
Is this final copy? I don't think developers will try to fail tests on purpose to view results. 

Prefer something like `You need to have failed tests` instead.

Reply:
I agree, i'll update and let @Adal3n3 know

Reply:
Approved by Adalene!